scripts/leaderboard.coffee

```diff
@@ -206,33 +206,37 @@ module.exports = (robot) ->
     msg.send response
 
   # response for adding to leaderboard
-  robot.respond /debut ([\w\-_]+)/i, (msg) ->
-    name = msg.match[1]
-    ScoreField = scorefield()
-    Aliases = aliases()
-    if verifyName(name, ScoreField, Aliases)
-      response = "#{name} is already in the game."
-      msg.send response
-    else
-      ScoreField[name.toLowerCase()] = 0
-      response = "Added #{name} to roster."
-      msg.send response
+  robot.respond /debut (.+)/i, (msg) ->
+    debut = msg.match[1].toLowerCase()
+    queries = debut.split " "
+    for name in queries
+      ScoreField = scorefield()
+      Aliases = aliases()
+      if verifyName(name, ScoreField, Aliases)
+        response = "#{name} is already in the game."
+        msg.send response
+      else
+        ScoreField[name.toLowerCase()] = 0
+        response = "Added #{name} to roster."
+        msg.send response
```

Comment:
Instead of sending a response for each user, a single response can be sent containing all the messages. For this, just move `msg.send` out of the loop and update msg for each query in loop.